Sources/Bodega/DiskStorage.swift

```diff
@@ -35,6 +35,26 @@ public actor DiskStorage {
     public func read(key: CacheKey, subdirectory: String? = nil) -> Data? {
         return try? Data(contentsOf: self.concatenatedPath(key: key.value, subdirectory: subdirectory))
     }
+    
+    /// Returns the modification date of the `CacheKey`, if it exists.
+    /// - Parameters:
+    ///   - key: A `CacheKey` for matching Data to a location on disk.
+    ///   - subdirectory: An optional subdirectory the caller can read from.
+    /// - Returns: The modification date of the data on disk if it exists, nil if there is no data stored behind the `CacheKey`.
+    public func modificationDate(key: CacheKey, subdirectory: String? = nil) -> Date? {
+        return try? self.concatenatedPath(key: key.value, subdirectory: subdirectory)
+            .resourceValues(forKeys: [.contentModificationDateKey]).contentModificationDate
+    }
+    
+    /// Returns the last access date of the `CacheKey`, if it exists.
+    /// - Parameters:
+    ///   - key: A `CacheKey` for matching Data to a location on disk.
+    ///   - subdirectory: An optional subdirectory the caller can read from.
+    /// - Returns: The last access date date of the data on disk if it exists, nil if there is no data stored behind the `CacheKey`.
```

Comment:
Small typo here, it says `date date` rather than `date`.

Reply:
Fixed